Xamarin.Essentials/Clipboard/Clipboard.mac.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Threading.Tasks;
+using AppKit;
+using Foundation;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Clipboard
+    {
+        static readonly string pasteboardType = NSPasteboard.NSPasteboardTypeString;
```

Comment:
See old comment on GetReadableTypesForPasteboard from Frank. ^

Reply:
Fixed this thanks! 👍